Rubberduck.Parsing/Symbols/ArgumentReference.cs

```diff
@@ -45,5 +49,39 @@ public class ArgumentReference : IdentifierReference
         public int NumberOfArguments { get; }
         public VBAParser.ArgumentListContext ArgumentListContext { get; }
         public Selection ArgumentListSelection { get; }
+
+        public override (string context, Selection highlight) HighligthSelection(ICodeModule module)
```

Comment:
I do not really like that this method is here. It solely deals with presentation concerns. This should rather be a method somewhere in `Core.UI`.